archetypes/blog.md

```diff
@@ -17,7 +17,7 @@ With contributions from secondary-author-name-1, ..., and secondary-author-n.
 
 ## Top-level heading
 
-Top-level headings start at **level 2**, as shown above.
+Top-level headings start at **level 2**. Your post should not include `# headings` but should instead only use `## headings`.
```

Comment:
```suggestion
Top-level headings start at **level 2**. This means, that your post should not include `# headings` for top-level headings but `## headings` instead.
```

The sentence _can_ lead to the assumption that we only accept level 2 headings, but of course they can use level3, level4, etc if they want to.